Command/Message/schema.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please review [contribution manual](https://bit.ly/contribution_manual), because properties are not documented properly. It is recommended to pass the [contribution checklist](https://smartdatamodels.org/index.php/contribution-checklist/)

Reply:
Added description in message schema